website/templates/emails/reviews_submission_confirmation.html.mako

```diff
@@ -11,9 +11,12 @@
         % if workflow == 'pre-moderation':
             <p>${reviewable.provider.name} has chosen to moderate their submissions using a pre-moderation workflow, which means your submission is pending until accepted by a moderator. You will receive a separate notification informing you of any status changes.</p>
         % elif workflow == 'post-moderation':
-            <p>${reviewable.provider.name} has chosen to moderate their submissions using a post-moderation workflow, which means your submission is public and discoverable, while still pending acceptance by a moderator. You will receive a separate notification informing you of any status changes.</p>
+            <p>${reviewable.provider.name} has chosen to moderate their submissions using a post-moderation workflow, which means your submission is public and discoverable, while still pending acceptance by a moderator.
```

Comment:
Stop me if I'm being too nitpicky, but if a tag contains newlines, I'd prefer the closing tag be on the same level as the opening tag, like
```html
<p>
    lots of text
    text text text
</p>
```
It seems much easier to read.